core/src/main/kotlin/com/malinskiy/marathon/test/Test.kt

```diff
@@ -6,7 +6,8 @@ data class Test(
     val pkg: String,
     val clazz: String,
     val method: String,
-    val metaProperties: Collection<MetaProperty>
+    val metaProperties: Collection<MetaProperty>,
+    val tags: Collection<String> = emptyList()
```

Comment:
MetaProprety is an abstraction over tags, annotations and so on. We should just reuse MetaProperty here

Reply:
Refer: 
[iOS]Added Support for Cucumberish Tags / Function Annotations https://github.com/MarathonLabs/marathon/pull/702